CC?=gcc in Nvim-R/R/nvimcom/src/apps/Makefile.win

Dear Sir,
After updating, I use `\rf` to lunch R terminal and find error massages:
```
* installing to library 'C:/R/library'
* installing *source* package 'nvimcom' ...
** libs
c:/Rtools/mingw_64/bin/gcc  -I"C:/R/include" -DNDEBUG       -D`C:/R/bin/R --slave -e 'cat(Sys.info()[[1]])'` -DLWS_NO_FORK  -DWIN32   -O2 -Wall  -std=gnu99 -mtune=generic -c nvimcom.c -o nvimcom.o
c:/Rtools/mingw_64/bin/gcc -shared -s -static-libgcc -o nvimcom.dll tmp.def nvimcom.o -lWs2_32 -LC:/R/bin/x64 -lR
(cd apps; make -f Makefile.win)
make[1]: Entering directory '/Users/ADMINI~1/AppData/Local/Temp/RtmpyWvysb/R.INSTALL350c47354a6/nvimcom/src/apps'
cc -m64 -mwindows -std=gnu99 -O3 -Wall -DWIN32 nclientserver.c -o nclientserver.exe -lWs2_32
make[1]: cc: Command not found
make[1]: *** [Makefile.win:10: nclientserver.exe] Error 127
make[1]: Leaving directory '/Users/ADMINI~1/AppData/Local/Temp/RtmpyWvysb/R.INSTALL350c47354a6/nvimcom/src/apps'
make: *** [Makevars.win:11: nvimapps] Error 2
ERROR: compilation failed for package 'nvimcom'
* removing 'C:/R/library/nvimcom'
* restoring previous 'C:/R/library/nvimcom'
In R CMD INSTALL
```
I have to change 
```
CC?=gcc
```
to 
```
CC=gcc
```
in `Nvim-R/R/nvimcom/src/apps/Makefile.win`. The installation of `nvimcom` is fine. 

What is this problem? By the way, I use windows 10.
Many thanks.

--- a/R/nvimcom/src/apps/Makefile.win
+++ b/R/nvimcom/src/apps/Makefile.win
@@ -1,4 +1,4 @@
-CC?=gcc
+CC ?=gcc
 ifeq "$(WIN)" "64"
     CFLAGS+=-m64 -mwindows -std=gnu99 -O3 -Wall -DWIN32
 else

